src/replication/adapter/quay/adapter.go

```diff
@@ -223,3 +257,35 @@ func (a *adapter) getNamespace(namespace string) (*model.Namespace, error) {
 		Name: namespace,
 	}, nil
 }
+
+// PushManifest ...
+func (a *adapter) PushManifest(repository, reference, mediaType string, payload []byte) error {
+	err := a.Adapter.PushManifest(repository, reference, mediaType, payload)
+	if err != nil {
+		if comErr, ok := err.(*common_http.Error); ok {
+			if comErr.Code == http.StatusAccepted {
+				return nil
+			}
+		}
+	}
+	return err
+}
```

Comment:
Maybe not need.

Reply:
In native PushManifest, it thinks 200 and 201 StatusCode as a successful response, but when I did replication between enterprise quay and harbor, it returns 202 StatusCode, so here maybe needs to do this to ignore 202 error.